README.md

```diff
@@ -14,10 +14,20 @@ Linux amd64 Tags
 
 | Tags                                       | Dockerfile                                                              | OS Version |
 |--------------------------------------------|-------------------------------------------------------------------------|------------|
-| `3.0`                                      | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet.Dockerfile)            | Debian 10  |
-| `3.0-slim`                                 | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-slim.Dockerfile)       | Debian 10  |
-| `3.0-appservice`, `3.0-dotnet3-appservice` | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-appservice.Dockerfile) | Debian 10  |
-| `3.0-dotnet3-core-tools` | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-core-tools.Dockerfile) | Debian 10  |
+| `3.0`                                      | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet.Dockerfile)            | Debian 10  |
+| `3.0-slim`                                 | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-slim.Dockerfile)       | Debian 10  |
+| `3.0-appservice`, `3.0-dotnet3-appservice` | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-appservice.Dockerfile) | Debian 10  |
+| `3.0-dotnet3-core-tools` | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-core-tools.Dockerfile) | Debian 10  |
+
+
+`mcr.microsoft.com/azure-functions/dotnet-isolated`
+
+| Tags                                       | Dockerfile                                                              | OS Version |
+|--------------------------------------------|-------------------------------------------------------------------------|------------|
+| `3.0`                                      | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-isolated/dotnet-isolated.Dockerfile)            | Debian 10  |
+| `3.0-dotnet-isolated5.0-slim`                                 | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-isolated/dotnet-slim.Dockerfile)       | Debian 10  |
+| `3.0-appservice`, `3.0-dotnet-isolated5.0-appservice` | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-isolated/dotnet-appservice.Dockerfile) | Debian 10  |
+| `3.0-dotnet-isolated5.0-core-tools` | [Dockerfile](host/3.0/buster/amd64/dotnet/dotnet-isolated/-isolated-core-tools.Dockerfile) | Debian 10  |
```

Comment:
Thanks for adding this!! :) Ahmed had asked me, but it slipped my mind.

Reply:
You are welcome!